packages/table/src/table-body.js

```diff
@@ -79,8 +137,6 @@ export default {
                 : ''
               ]
             ).concat(
-              this._self.$parent.$slots.append
```

Comment:
@liril-net @Leopoldthecoder Any particular reason this slot was moved? It breaks the infinite loading plugins for the table: https://jsfiddle.net/4ez4j2j4/ (try removing the `@next` in HTML section line 2 to see it working).

Reply:
`append` slot should be unique in one `el-table` to not affect `render` in `el-tbody`, so i move it out of `el-tbody`.